### PR TITLE
Hiding refresh list on non-Mainnet networks

### PR DESCRIPTION
**Pull request by NiranjanaBinoy**
Fixes no 7, in [10.2.0 RC issues doc](https://docs.google.com/document/d/1BiTKWFKhYHRx697RrEs8Xz7R2qj0aZSN58h0JSsoyU0/edit#heading=h.lwrpcld9iufi)

The token detection is currently available only on Mainnet, so the `refresh list` link has to be displayed on Mainnet only.

### CODE REVIEW

File: ui/components/app/import-token-link/import-token-link.component.js [hunk 1]
```diff
@@ -21,14 +22,16 @@ export default function ImportTokenLink() {
...
+      {isMainnet && (
+        <Button
+          className="import-token-link__link"
+          type="link"
+          onClick={() => detectNewTokens()}
+        >
+          {t('refreshList')}
+        </Button>
+      )}
+      {isMainnet && ' or '}
```
Comment (ryanml) [inline ui/components/app/import-token-link/import-token-link.component.js:34]: Looks like the `or` should be localized?

Comment (jpuri) [reply ui/components/app/import-token-link/import-token-link.component.js:34]: Also repeating `isMainnet && ` can be avoided.

File: ui/components/app/import-token-link/import-token-link.component.js [hunk 2]
```diff
@@ -37,8 +40,15 @@ export default function ImportTokenLink() {
...
+          : t('importTokens').charAt(0).toUpperCase() +
+            t('importTokens').slice(1)}
       </Button>
     </Box>
   );
 }
+
+ImportTokenLink.propTypes = {
+  isMainnet: PropTypes.bool,
+};
```
Comment (jpuri) [inline ui/components/app/import-token-link/import-token-link.component.js:54]: `isMainnet` can be given a default value also.

File: ui/components/app/import-token-link/import-token-link.component.js [hunk 3]
```diff
@@ -37,8 +42,15 @@ export default function ImportTokenLink() {
           addTokenEvent();
         }}
       >
-        {t('importTokens')}
+        {isMainnet
+          ? t('importTokens')
```
Comment (ryanml) [inline ui/components/app/import-token-link/import-token-link.component.js:46]: Any reason we can't just use `t('importTokensCamelCase')` here for non-mainnet networks??

**Review by jpuri — APPROVED** [review]

File: ui/components/app/import-token-link/import-token-link.component.js (hunk 3 above)
Comment (NiranjanaBinoy) [reply ui/components/app/import-token-link/import-token-link.component.js:46]: The message in `t('importTokensCamelCase')` is `Import Token` but we need `Import token` here

Comment (ryanml) [reply ui/components/app/import-token-link/import-token-link.component.js:46]: or if need be can we add a new translation called `'importTokensLabel'` that equals `"Import tokens"`? I now see it's for consistency with `refreshList`

Comment (NiranjanaBinoy) [reply ui/components/app/import-token-link/import-token-link.component.js:46]: Yes, but I was not sure if it is worth adding multiple entries for the same message just for case change, if I add this one that will be 3rd entry for the same message.

**Review by ryanml — APPROVED** [review]

**Pull request merged**